WooCommerce/Classes/ViewRelated/Editor/AztecEditorView.swift

```diff
@@ -0,0 +1,21 @@
+import SwiftUI
+
+/// `UIViewControllerRepresentable` to use an Aztec Editor in SwiftUI.
+/// This view needs to be wrapped inside a parent View that manages its states and toolbar as needed.
+/// Params:
+/// - value: Binding to the content of the editor.
+/// - viewController: To get the content of the editor, the parent View can pass this and then call `getLatestContent()`
+///
+struct AztecEditorView: UIViewControllerRepresentable {
+    @Binding var value: String
+    @Binding var viewController: AztecEditorViewController?
```

Comment:
As discussed on Slack, it's hacky to set the view controller as `@Binding` because bindings are supposed to be used for properties that are subject to changes.

Instead, you should get the value of the html content from the editor and set it to the `value` binding. My suggestion:
- Create a coordinator for the AztecEditorView.
- Set the delegate for `htmlTextView` of `AztecEditorViewController.editorView` to this coordinator.
- In the `textViewDidChange` method of the `UITextViewDelegate` conformance, set the `text` binding to the updated text similar to [this](https://github.com/shaps80/SwiftUIBackports/blob/f5f23b016eeda6642a0fe1020241af19c9c05556/Sources/SwiftUIBackports/iOS/TextEditor/TextEditor.swift#L130).

Let me know if you have any questions.